style.css

```diff
@@ -0,0 +1,137 @@
+body{
+    margin: 500;
+}
+
+.headerBar{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 50px;
+    }
+
+#searchBar{
+    width: 550px;
+    height: 25px;
+}
+::placeholder {
+   padding-left: 10px;
+   
+  }
+#searchButton{
+    float: right;
+    width: 70px;
+    height: 31px;
+    
+}
+.logoMenu{
+    margin: 20px;
+    height : 20px; 
+    width : auto;
+}
+.logoYoutube {
+    
+    height : 30px; 
+    width : 100px;
+}
+.logos{
+    display: flex;
+    align-items: center;
+}
+.list{
+    list-style-type: none;
+}
+.video{
+    display: inline-block;
+    justify-content: center;
+    text-align: left;
+    margin-left: 120px;
+    position: absolute;
+    
+}
+.videoTittle {
+    
+    font-family: Roboto, Ariel, sans-serif;
+    color: #030303;
+    font-size: 18px;
+    font-weight: 400px;
+    margin-top: 10px;
+}
+.videoIcons{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    font-family: Roboto, Ariel, sans-serif;
+    color: #636360;
+    font-size: 14px;
+}
+.infoUser{
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    
+
+}
+
+.description{
+    display: flex;
+    font-size: 14px;
+    font-family: Roboto, Ariel, sans-serif;
+    color: #030303;
+    margin-bottom: 400px;
+    padding-bottom: 0%;    
+}
+.user {
+    display: flex;
+    font-size: 13px;
+    font-family: Roboto, Ariel, sans-serif;
+    color: #606060
+}
+.userName{
+    color: #030303;
+    font-family: Roboto, Ariel, sans-serif;
+    font-size: 14px;
+    font-weight: bold;
+}
+.subscribe input{
+    font-size: 20px;
+    font-family: Roboto, Ariel, sans-serif;
+    background-color: #CC0000;
+    margin: 0px 4px;
+    padding: 10px 16px;
+    color: white;
+}
+.recommended {
+    float: right;
+}
+.video1{
+    display: flex;
+    align-items: center;
+    margin: 20px;
+}
+.video1Text{
+    margin: 20px;
+    font-size: 13px;
+    font-family: Roboto, Ariel, sans-serif;
+    color: #606060
+}
+.subscribe{
+    margin-bottom: 500px;
+}
+.subscribers{
+    margin: 0px;
+}
+#icon1{
+    margin: 20px;
+}
+#icon2{
+    margin: 20px;
+    
+}
+#icon3{
+    margin: 20px;
+    
+}
+#icon4{
+    margin: 20px;
+    
+}
```

Comment:
There is a lot of repetition going on here. I'd best advice the good use of a class to style up all the similar styling or bundle the id selectors into one and style them all in one line. It is not only a good practice but it makes your code looks neater.